packages/common-ethereum/src/project/versioned/v1_0_0/model.ts

```diff
@@ -95,6 +98,14 @@ export class ProjectNetworkV1_0_0 extends ProjectNetworkDeploymentV1_0_0 {
 }
 
 export class DeploymentV1_0_0 {
+  @IsString()
+  name: string;
+  @IsString()
+  version: string;
+  @IsString()
+  description: string;
+  @IsString()
+  repository: string;
```

Comment:
Can be removed too

Reply:
To fix the issue in https://github.com/subquery/subql/issues/1478 I added the `forbidNonWhitelisted` and `forbidUnknownValues` option to manifest validation:
https://github.com/subquery/subql-ethereum/blob/3d511d8be526a31bcd3d5f0fb8a62ed6fdb7ad7b/packages/common-ethereum/src/project/versioned/v1_0_0/model.ts#L206

So, the validation will throw an error without these fields

Reply:
The changes on 206 cant be right then. When `subq publish` is use we create a deployment which is a subset of the manifest and will not have these fields